content/100-getting-started/02-setup-prisma/01-add-to-existing-project.mdx

```diff
@@ -480,7 +480,7 @@ Notice that the [`@prisma/client` node module](../../concepts/components/prisma-
 npx prisma generate
 ```
 
-This command reads your Prisma schema and generates your Prisma Client library into `node_modules/@prisma/client`.
+This command reads your Prisma schema and generates your Prisma Client library into `node_modules/.prisma/client`.
```

Comment:
```suggestion
This command reads your Prisma schema and generates your Prisma Client library:
```

The folder is mentioned both in the section before and after this one, so not necessary here.